docs/ReferenceArrayInput.md

```diff
@@ -328,6 +330,76 @@ Then to display a selector for the post tags, you should call `<ReferenceArrayIn
 <ReferenceArrayInput source="tags_ids" reference="tags" />
 ```
 
+## `readOnly`
+
+If `true`, the default input is in read-only mode and the user can't change the value.
+
+{% raw %}
```

Comment:
<!-- Polish comments are like a suggestion, where there is nothing necessarily wrong with the relevant content, there's just some ways to immediately improve the quality.  -->
**polish:** ‏ `{% raw %}` are not necessary when there is no objects inside the snippet. We should remove them when they are not needed.